.github/workflows/ci.yml

```
@@ -27,4 +27,6 @@ jobs:
       - run: npm ci
       - name: Check TypeScript
         run: npx tsc --noEmit
+      - name: Run Biome
+        run: npx biome ci .  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)